fetcher/utils.go

```diff
@@ -49,3 +50,12 @@ func tryAgain(fetchMsg string, thisBackoff backoff.BackOff, err error) bool {
 
 	return true
 }
+
+// checkError compares a *fetcher.Error to a simple type error and returns
+// a boolean indicating if they are equivalent
+func checkError(fetcherErr *Error, err error) bool {
```

Comment:
Can we add some tests for this method?

Reply:
I guess this is tested implicitly through other existing tests. so should be fine